src/wallet/rpcwallet.cpp

```diff
@@ -3603,7 +3607,9 @@ UniValue z_getbalance(const UniValue& params, bool fHelp)
     if (fromTaddr) {
         nBalance = getBalanceTaddr(fromaddress, nMinDepth, false);
     } else {
-        nBalance = getBalanceZaddr(fromaddress, nMinDepth, INT_MAX, false);
+        // TODO: Return an error if a UA is provided (once we support UAs).
```

Comment:
How would you get the balance of an Orchard address, other than by passing a UA containing it?

Reply:
We agreed to deprecate `z_getbalance` because it only returns a single value: https://github.com/zcash/zcash/issues/5191